integrations/google_vertex/src/haystack_integrations/components/embedders/google_vertex/document_embedder.py

```diff
@@ -96,6 +96,10 @@ def __init__(
 
         :raises ValueError: If the provided model is not in the list of supported models.
         """
+        soft_deprecation_msg = (
+            "This component uses a deprecated SDK. We recommend using the GoogleGenAIDocumentEmbedder instead."
```

Comment:
Could we also add to the message how to access and install the `GoogleGenAIDocumentEmbedder`? I'd either link to the docs for Google GenAI or give the name of the package.

Reply:
added link to the docs